output/watermark-to-season.json

```diff
@@ -3,6 +3,7 @@
   "/common/destiny2_content/icons/dd71a9a48c4303fd8546433d63e46cc7.png": 1,
   "/common/destiny2_content/icons/591f14483308beaad3278c3cd397e284.png": 2,
   "/common/destiny2_content/icons/50d36366595897d49b5d33e101c8fd07.png": 2,
+  "/common/destiny2_content/icons/5f5bed34dcd062be8302ce27b542dce9.png": 3,
```

Comment:
This should not be here... assistance would be appreciated...

Reply:
https://data.destinysets.com/i/InventoryItem:109666087

`iconWatermark` and `quality.displayVersionWatermark[0]` did not match...